external/README_PROTOCOL.md

```diff
@@ -0,0 +1,57 @@
+# Network Protocols in External libraries
+
+## Contents
+
+> [Network Applications](#network-applications)  
+> [How to enable](#how-to-enable)
+
+## Network Applications
+
+This directory contains most of the network applications.  
+include:
+
+    dhcpc     - Dynamic Host Configuration Protocol (DHCP) client.
+                See external/include/protocols/dhcpc.h for interface information.
+    dhcpd     - Dynamic Host Configuration Protocol (DHCP) server.
+                See external/include/protocols/dhcpd.h for interface information.
+    ftpc      - FTP client.
+                See external/include/protocols/ftpc.h for interface information.
+    ftpd      - FTP server.
+                See external/include/protocols/ftpd.h for interface information.
+    mdns      - multicast Domain Name System (mDNS).
+                See external/mdns/mdns.h for interface information.
+    netlib    - network libraries that includes IP address support, HTTP support and generic server logic.
```

Comment:
it would be better to delete this comment 'HTTP support and generic server logic'